docs/cluster-configuration.md

```diff
@@ -343,3 +343,97 @@ policies:
             '{{hub $configMap:=(lookup "v1" "ConfigMap" "" (printf "%s-pg" .ManagedClusterName)) hub}}{{hub dig "data" "install-plan-approval" "Manual" $configMap hub}}'
     - path: source-crs/LcaSubscriptionOperGroup.yaml
 ```
+
+### Updating the ClusterTemplate schemas
+
+We assume a ManagedCluster has been installed through a `ProvisioningRequest` referencing the [sno-ran-du.v4-Y-Z-3](samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-v4-Y-Z-3.yaml) `ClusterTemplate` CR.
+
+In this example we are updating the policy template schema - `spec.templateParameterSchema.policyTemplateParameters`. This update means that the ACM PG requires extra configuration values.
+We assume we are starting from the [sno-ran-du-pg-v4-Y-Z-v2](samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-pg-v4-Y-Z-v2.yaml) ACM PG, but want to add `SriovNetworkNodePolicy` configuration.
+
+The following steps need to be taken:
+1. Upversion the cluster template content:
+    * A new ACM PG is created - [sno-ran-du-pg-v4-Y-Z-v3](samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-pg-v4-Y-Z-v3.yaml):
+        * `metadata.name` is updated from `sno-ran-du-pg-v4-Y-Z-v2` to `sno-ran-du-pg-v4-Y-Z-v3` (the `ztp-sno-ran-du-v4-Y-Z` namespace is kept).
+        * `policyDefaults.placement.labelSelector.sno-ran-du-policy` is updated from `v2` to `v3` such that the policy binding is updated.
+        * All policy names are updated from `v2` to `v3` (example: `v2-subscriptions-policy` -> `v3-subscriptions-policy`).
+        * The following manifests are added under the `v3-subscriptions-policy`:
+        ```yaml
+        - path: source-crs/SriovNetworkNodePolicy-SetSelector.yaml
```

Comment:
I just noticed that the SriovNetworkNodePolicy CRs for du_fh and du_mh are missing in the initial PolicyGenerator (PG), but their SriovNetworks are included. Based on the legacy ZTP, they are all part of the DU profile. I think we should include both of them in the Day 0 PG, so that the VFs of the configured interfaces can be created and attached to the application.

For day2 schema update usecase, maybe we can add a new pair of SriovNetwork and SriovNetworkNodePolicy?